components/speedreader/speedreader_rewriter_service.h

```diff
@@ -43,15 +42,11 @@ class SpeedreaderRewriterService : public SpeedreaderComponent::Observer {
       delete;
 
   // The API
-  bool IsWhitelisted(const GURL& url);
+  bool URLLooksReadable(const GURL& url);
   std::unique_ptr<Rewriter> MakeRewriter(const GURL& url);
   const std::string& GetContentStylesheet();
 
  private:
-  using GetDATFileDataResult =
-      brave_component_updater::LoadDATFileDataResult<speedreader::SpeedReader>;
-
-  void OnLoadDATFileData(GetDATFileDataResult result);
   void OnLoadStylesheet(std::string stylesheet);
 
   // Default backend is an Arc90 implementation.
```

Comment:
we probably don't need `backend_` now because it's always constant

Reply:
Fixed in https://github.com/brave/brave-core/pull/10090/commits/bab3ce896441e392eb05e54d46c2133dcd5b5a60